### PR TITLE
Linkedin,Facebook links updated inReadme file of  mattermost-server

### PR DESCRIPTION
**Pull request by drraghavendra**
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: DRY up the codebase [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Welcome new contributors!
```
Comment (hanzei) [inline DRY up the codebase:1]: Would you please remove these unrelated changes?

**Review by hanzei — CHANGES_REQUESTED** [review]

**Review by hanzei — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 2]
```diff
@@ -77,6 +77,8 @@ Receive notifications of critical security updates. The sophistication of online
 
 - **Twitter** - Follow [Mattermost](https://twitter.com/mattermost)
 - **Blog** - Get the latest updates from the [Mattermost blog](https://mattermost.com/blog/).
+- **facebook** - Follow [Mattermost](https://www.facebook.com/MattermostHQ)
+- **linkedin** - Follow [Mattermost](https://www.linkedin.com/company/mattermost/)
```
Comment (jfrerich) [inline README.md:81]:
```suggestion
- **Facebook** - Follow [Mattermost](https://www.facebook.com/MattermostHQ)
- **Linkedin** - Follow [Mattermost](https://www.linkedin.com/company/mattermost/)
```

4/5 suggest we capitalize the first letters, to match the other listings.  cc @jasonblais what do you think?

**Review by jasonblais — APPROVED** [review]
I updated the capitalization, approving. Thank you for the contribution!

**Comment by jasonblais** [comment]
@drraghavendra thank you for the contribution!

**Comment by drraghavendra** [comment]
sir @jasonblais PR is   hacktoberfest accepted check details here  https://github.com/mattermost/mattermost-webapp/pull/6916

**Comment by jasonblais** [comment]
Thanks, I added the `hacktoberfest-accepted` label :+1:

**Comment by drraghavendra** [comment]
Respected Sir @jasonblais as per the inference provided earlier,I guess Goodies are to provided for this Pull Request.

**Comment by jasonblais** [comment]
Hi @drraghavendra, if you didn't receive a thank you gift from us for your contribution, please email us at community[at]mattermost.com. It is possible it didn't reach you due to shipping delays over the pandemic.

**Pull request merged**